src/functions/module_lookup.ts

```diff
@@ -262,11 +280,14 @@ export const MATCH: AddFunctionDescription = {
 
 export const ROW: AddFunctionDescription = {
   description: _lt("Row number of a specified cell."),
-  args: args(
-    `cell_reference (meta, default=${_lt(
-      "The cell in which the formula is entered by default"
-    )}) ${_lt("The cell whose row number will be returned.")}`
-  ),
+  args: [
+    arg(
+      "cell_reference (meta, default='this cell')",
+      _lt(
+        "The cell whose row number will be returned. By default, this function uses the cell in which the formula is entered by default"
```

Comment:
"by default" is present twice in the sentence.
Either keep the first one or second one